back/spec/acceptance/users_spec.rb

```diff
@@ -320,7 +320,7 @@
         parameter :sort, "Sort user by 'created_at', '-created_at', 'last_name', '-last_name', 'email', '-email', 'role', '-role'", required: false
         parameter :group, 'Filter by group_id', required: false
         parameter :can_moderate_project, 'Filter by users (and admins) who can moderate the project (by id)', required: false
-        parameter :can_moderate, 'Filter out admins and moderators', required: false
+        parameter :blocked, 'Return only blocked users', required: false
```

Comment:
why are you removing `can_moderate` parameter here?

Reply:
oops. good catch

Reply:
I put it back 😬